src/components/auth/torus/sdk/TorusSDK.js

```diff
@@ -30,14 +30,18 @@ class TorusSDK {
   }
 
   constructor(config, logger) {
-    const { env, torusProxyContract, torusNetwork } = config
+    const { env, publicUrl, torusProxyContract, torusNetwork, torusUxMode = 'popup' } = config
+    const baseUrl = torusUxMode === 'popup' ? `${publicUrl}/torus` : publicUrl
```

Comment:
```suggestion
    const popupMode = torusUxMode === 'popup'
    const baseUrl = publicUrl + (popupMode ? '/torus' : '')
    const redirectPath = popupMode ? 'redirect' : 'Welcome/Auth'
```

Reply:
As we only 2 states, we could use just one boolean instance var to keep the mode we running.
This allows us to do string comparison once , inside TorusSDK constructor.
Then we'll just checking torusSDK.popupMode, which is shorted in the code and faster as uses simple bitwise operation